index.html

```diff
@@ -0,0 +1,99 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+<!DOCTYPE html>
+<head>
+  <h1 id="title">Amazon Survey Form</form></h1>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body id=".left">
+  <title id="title">
+    <h1>Amazon Survey Form</h1></title>
+<header id="main-header" class="main-header"></header>
+ 
+    <p id="description"> Personal Information, Please<p>
+<main>
+     
```

Comment:
In my opinion, it would look more beautiful and readable if you remove empty line spaces from your code.